perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -850,4 +850,14 @@ sub admin_stats {
     return 1;
 }
 
+sub problem_confirm_email_extras {
+    my ($self, $c, $report) = @_;
```

Comment:
I think C is $self->{c} here, so you shouldn't need to pass it in - do check (and by that, I mean write a test to show the email is being correctly written in the two cases :) ).